browser/ui/content_settings/brave_content_setting_image_models.cc

```diff
@@ -1,4 +1,5 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
Was this change announced somewhere? A think, if it was then we should fix all sources at once, otherwise its better to fix the linter. @bbondy could you please clarify?

Reply:
@iefremov it is enforced by the linter. There was a discussion in `#brave-core`

Reply:
@bridiver maybe we can fix the whole codebase by a single PR then? I can do one